gdx/src/com/badlogic/gdx/graphics/g3d/loader/ObjLoader.java

```diff
@@ -243,9 +243,10 @@ protected ModelData loadModelData (FileHandle file, boolean flipV) {
 			if (hasNorms) attributes.add(new VertexAttribute(Usage.Normal, 3, ShaderProgram.NORMAL_ATTRIBUTE));
 			if (hasUVs) attributes.add(new VertexAttribute(Usage.TextureCoordinates, 2, ShaderProgram.TEXCOORD_ATTRIBUTE + "0"));
 
-			String nodeId = "node" + (++id);
-			String meshId = "mesh" + id;
-			String partId = "part" + id;
+			String stringId = "" + (++id);
```

Comment:
`Integer.toString()` will be probably easier to read and more efficient, but that is just a nitpick.
Also for lines below, it is usually better to call `"default".equals(name)` instead of `name.equals("default")` because it prevents a NPE. Again, just a minor detail.